engine/overworld/sprite_collisions.asm

```diff
@@ -296,7 +296,7 @@ DetectCollisionBetweenSprites:
 ; to indicate which sprite the collision occurred with
 	inc l
 	inc l
-	ldh a, [hFF8F] ; a = loop counter
+	ldh a, [hSpriteCollisionsLoopCounter]
 	ld de, SpriteCollisionBitTable
```

Comment:
The data for `SpriteCollisionBitTable` should use a loop:

```
SpriteCollisionBitTable:
FOR n, 16
	bigdw 1 << n
ENDR
```

(The `bigdw` macro will need porting from pokecrystal.)